src/main/java/frc/robot/RobotContainer.java

```diff
@@ -7,16 +7,28 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.subsystems.pivot.Pivot;
+import frc.robot.subsystems.pivot.Pivot.PivotGoal;
+import frc.robot.subsystems.pivot.PivotConstants;
+import frc.robot.subsystems.pivot.PivotKrakenHardware;
 
 public class RobotContainer {
+  private final Pivot kPivot =
+      new Pivot(new PivotKrakenHardware(PivotConstants.kMotorConfiguration));
+
   private final CommandXboxController kPilotController =
       new CommandXboxController(Constants.kPilotControllerPort);
 
   public RobotContainer() {
     configureBindings();
   }
 
-  private void configureBindings() {}
```

Comment:
In the trunk we'll want to put the code block brackets on their own lines so that the templates can insert their lines between the curly brackets.